constructors/bq/sql/VERSION.sql

```diff
@@ -0,0 +1,13 @@
+-----------------------------------------------------------------------
+--
+-- Copyright (C) 2021 CARTO
+--
+-----------------------------------------------------------------------
+
+CREATE OR REPLACE FUNCTION `@@BQ_PROJECTID@@.@@BQ_DATASET_CONSTRUCTORS@@.VERSION`()
+    RETURNS STRING
+    DETERMINISTIC
+    LANGUAGE js
+AS """
+    return '1.0.0';
```

Comment:
Sorry for the noise, I misplace a comment about this: this is where I thing the JS version function should be used. (I'll take a look a view at the SETool PR, because I think it should handle this.

Reply:
Ok, no problem!